### ISSUE TITLE
Sidebar: after dragging feed to iCloud account, there’s a delay before it shows up

### ISSUE THREAD

**Issue by brentsimmons** [Ranchero-Software/NetNewsWire#2725]
It should appear right away, but it appears as if the feed is read before it actually appears in the sidebar.

**Comment by brentsimmons** [comment]
Sometimes it doesn’t show up until quitting and relaunching the app. (Seems to be more true when dragging a folder?)

**Comment by vincode-io** [comment]
We always have always read a feed before adding it for all account types.  I assume this is like #2728 where you don't want to validate feeds on copying anymore?

**Comment by brentsimmons** [comment]
Would this be possible? When dragging a feed in the sidebar from any account to the iCloud account, it just gets added/accepted immediately.

Same when dragging to a local account. (But Feedly, Feedbin, etc. should retain their current behavior.)

**Comment by brentsimmons** [comment]
But — importantly — when adding a feed from outside the sidebar, the standard validation process applies.

**Comment by vincode-io** [comment]
I can remove the validation, but it isn't a straight forward change. We will have to add API to the `Account` and `AccountDelegate` to differentiate between local and non-local `createWebFeed` calls.

I don't think this will accomplish what you want, which is an instant add of the feed when copying/moving it. That's because we always try to add it to the sync service first before adding it locally so that we can either not add or rollback the locally created feed.  That's so that if we crash or get force quit before adding it to the service, we aren't out of sync with the service.  All the sync services work fundamentally the same way when adding a moving/copying a feed.

To add the feed before doing the network call would require us to do something for manipulating feeds durably.  We would need to add some kind of journal database like we have for unread counts (the SyncDatabase) and then build a process that keeps retrying to add the feed until successful.

In summary, removing feed validation will speed up the move/copy process, but unless we do major surgery, there is still going to be network lag that impacts the experience.
